### PR TITLE
Init test condition

### PR DESCRIPTION
**Pull request by pabloopez**
# Instruqt automation: `init.sh` aware of being run within a `instruqt track test`

This PR is the first step to enable automated tests for Sysdig labs at instruqt.

This is a WIP. One of the migrated labs (knps lab) already includes some basic scripts in place and I want to start testing in parallel this feature and review that all test are green. Consider that this is not the final/desired state of this and there are some blockers and things to improve:
- Instruqt has not implemented a way to pass ENV_VARS to the `instruqt test track` command yet. The FR request is done but is not implemented yet. To work around this problem, current keys are base64 encoded and decoded when used. Next iteration will store the keys as GH Secrets for our automation or just as ENV_VARS when running test locally.
- It removes nginx proxy, as we do not have a way to directly test the embeded Sysdig Dashboards UI. 
- Another limitation: the existing test account is linked to a particular region. When extending this tests other regions should be considered. But right now, it is enough to enable basic testing.

I request to merge this to main, as it is working fine with no errors and it does not influence normal execution of the`init.sh` job. The approach is not the best one, as it just replaces every request for user input (`read -p "message" VAR`) with a condition to check for a test-aware variable.

### CODE REVIEW

**Comment by pabloopez** [comment]
This is not required in any way, but if any of you is interested in accessing the test account, you can use:
`https://us2.app.sysdig.com/`
`team-training+instruqt@sysdig.com`

And ask for bitwarden/lastpass key. Or Slack, I don't mind as this is just for testing.

File: common/prepare-track/init.sh [hunk 1]
```diff
@@ -96,9 +101,14 @@ function set_values () {
     PROMETHEUS_ENDPOINT=$MONITOR_URL'/prometheus'
 
     # Tabs url_redirect
-    sed -i -e "s@_MONITOR_URL_@$MONITOR_URL@g" /etc/nginx/nginx.conf
-    sed -i -e "s@_SECURE_URL_@$SECURE_URL@g" /etc/nginx/nginx.conf
-    systemctl restart nginx
+    if [[ ${INSTRUQT_USER_ID} == "testuser-"* ]]; 
+    then
+        systemctl stop nginx
```
[the remark this answers is not in the document]
Comment (pmusa) [reply common/prepare-track/init.sh:106]: I mean, what is the impact if we don't have this if statement?

**Review by pmusa — CHANGES_REQUESTED** [review]
- The .DS_Store file should be removed from the PR. I suggest we also include a line to the `.gitignore` file to remove them in the future.

The rest looks good to me. I left a minor comment.

File: common/prepare-track/init.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (pabloopez) [reply common/prepare-track/init.sh:106]: No impact at all if we leave it as it is. 

The reason to kill the nginx service is that in *test* mode there's no way to check what's in the tabs.

[the remark this answers is not in the document]
Comment (pmusa) [reply common/prepare-track/init.sh:106]: I don't see an advantage in killing nginx. I would keep it.

[the remark this answers is not in the document]
Comment (pabloopez) [reply common/prepare-track/init.sh:106]: Neither I do in keeping it there, but let's keep it. It is not a big deal.

[the remark this answers is not in the document]
Comment (pabloopez) [reply common/prepare-track/init.sh:106]: https://github.com/draios/instruqt-assets/pull/38

**Pull request merged**